src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceSpecCallbackHandler.java

```diff
@@ -493,13 +499,33 @@ && isOneVOne(caseData)) {
 
         builder.businessProcess(businessProcess);
 
+        assembleFRCDocuments(caseData, builder);
+
         return AboutToStartOrSubmitCallbackResponse.builder()
             .data(builder.build().toMap(objectMapper))
             .state(nextState)
             .build();
 
     }
 
+    private void assembleFRCDocuments(CaseData caseData, CaseData.CaseDataBuilder<?, ?> updatedCaseData) {
```

Comment:
the assembleFRCDocuments can be service?
because I saw that we have in 3 different handlers the same method with same propose but different data